spec/features/chitter_spec.rb

```diff
@@ -0,0 +1,16 @@
+feature 'Viewing chitter' do
+    scenario 'visiting the chitter page' do
+        visit('/chitter')
+        expect(page).to have_content("Chitter feed")
+    end
+end
+
+feature 'Posting a peep' do
+    scenario 'visiting the page' do
+        visit('post-a-peep')
```

Comment:
Does this work without the '/' before the page name (post-a-peep)? I've just tried running `rspec` to check and your tests pass. So maybe '/' is not always needed. Interesting to know!